uma/protocol/payreq.py

```diff
@@ -118,8 +118,8 @@ def from_json(cls: "type[PayRequest]", json_encoded: str) -> "PayRequest":
         is_amount_string = "amount" in json_dict and isinstance(
             json_dict["amount"], str
         )
-        is_uma = "payerData" in json_dict and "compliance" in json_dict["payerData"]
-        is_v1 = ("convert" in json_dict) and is_uma
+        is_uma = json_dict.get("payerData") and "compliance" in json_dict["payerData"]
```

Comment:
LGTM, but just for my own understanding, what's the difference?

Reply:
Assuming the payload has `"payer_data": None`
- `json_dict.get("payerData")=None` and `bool(None)=False`, so the second part `"compliance" in json_dict["payerData"]` won't be executed
- `"payerData" in json_dict = True`, and it throws in second part `"compliance" in None` because `argument of type 'NoneType' is not iterable`

Reply:
`"payerData" in json_dict` basically checks if the dictionary has the keyword or not, but doesn't care what is the corresponding value

Reply:
ahhh I see, thanks for the clarification!